pkg/model/estimator/local/ratio_test.go

```diff
@@ -61,17 +64,17 @@ var _ = Describe("Test Ratio Unit", func() {
 			Pkg:    20,
 		}
 		nodeMetrics.AddNodeComponentsEnergy(componentsEnergies)
-		Expect(nodeMetrics.EnergyInCore.Curr()).Should(BeEquivalentTo(10))
-		Expect(nodeMetrics.EnergyInDRAM.Curr()).Should(BeEquivalentTo(3))
-		Expect(nodeMetrics.EnergyInUncore.Curr()).Should(BeEquivalentTo(2))
-		Expect(nodeMetrics.EnergyInPkg.Curr()).Should(BeEquivalentTo(20))
+		Expect(nodeMetrics.EnergyInCore.Aggr()).Should(BeEquivalentTo(10))
+		Expect(nodeMetrics.EnergyInDRAM.Aggr()).Should(BeEquivalentTo(3))
+		Expect(nodeMetrics.EnergyInUncore.Aggr()).Should(BeEquivalentTo(2))
+		Expect(nodeMetrics.EnergyInPkg.Aggr()).Should(BeEquivalentTo(20))
 
 		nodePlatformEnergy := map[string]float64{}
 		nodePlatformEnergy["sensor0"] = 40
 		nodeMetrics.AddLastestPlatformEnergy(nodePlatformEnergy) // must be higher than components energy
-		Expect(nodeMetrics.EnergyInPlatform.Curr()).Should(BeEquivalentTo(40))
+		Expect(nodeMetrics.EnergyInPlatform.Aggr()).Should(BeEquivalentTo(40))
 
 		UpdateContainerEnergyByRatioPowerModel(containersMetrics, nodeMetrics)
-		Expect(containersMetrics["containerA"].EnergyInPkg).Should(BeEquivalentTo(10))
+		Expect(containersMetrics["containerA"].EnergyInPkg.Curr).Should(BeEquivalentTo(10))
```

Comment:
@sunya-ch the changes in `stats.go` is needed to pass this test

Reply:
Curr -> Curr()

Reply:
Please fix here before merge.

Reply:
this is a `UInt64Stat`, the `Curr()` function is available in `UInt64StatCollection`

Reply:
Sorry. I also miss that. EnergyInPkg is not a collection.